tests/network-tests/src/nicaea/dto/fillOpeningParameters.ts

```diff
@@ -0,0 +1,79 @@
+import BN from 'bn.js';
+
+export class FillOpeningParameters {
+  private amountPerPayout!: BN;
```

Comment:
With ! we asserting the value will be assigned before it will be used. As alternative, we could initialize the values with default values, but the object is not intended to be used without all fields being initialized.

Reply:
ok thanks for clarification. So we are infact using the `!` incorrectly, the fields should then be initialized in the constructor. No ?

Reply:
With initialization in the constructor, the assignment assertion will be excessive (since the valued are guaranteed to be initialized). So, when the values will be initialized in the constructor, ! will be removed.